packages/sit-onyx/src/components/examples/GridPlayground/GridPlayground.vue

```diff
@@ -1,167 +1,275 @@
-};
+const currentBreakpoint = computed(() => {
+  const breakpoint = Object.entries(ONYX_BREAKPOINTS).reduce((prev, [name, width]) => {
+    if (viewportSize.width.value >= width) return name;
+    return prev;
+  }, "2xs");
 
-const handleAddModifier = () => {
-  selectedElement.value?.push({ spans: 2 });
-};
+  return `${breakpoint} (${Math.round(viewportSize.width.value)}px)`;
+});
 </script>
 
 <template>
-  <div class="onyx-grid-playground">
-    <form @submit.prevent>
-      <fieldset class="onyx-grid-playground__settings">
-        <legend>Grid options</legend>
-        <label><input v-model="isCentered" type="checkbox" />Centered</label>
-        <label><input v-model="is20Xl" type="checkbox" />XL with 20 columns</label>
-        <label><input v-model="isMaxMd" type="checkbox" />Max md width</label>
-        <label><input v-model="isMaxLg" type="checkbox" />Max lg width</label>
-      </fieldset>
-      <fieldset class="onyx-grid-playground__settings">
-        <legend>Selected Grid Element options</legend>
-        <template v-if="selectedElement">
-          <button type="button" @click="deleteElement()">Delete Grid Element</button>
-          <div v-for="(props, i) in selectedElement" :key="i">
-            <label>
-              Breakpoint:
-              <select v-model="props.breakpoint">
-                <option :value="undefined">None</option>
-                <option v-for="bp in Object.keys(BREAKPOINTS)" :key="bp" :value="bp">
-                  {{ bp }}
-                </option>
-              </select>
-            </label>
-            <label>
-              Columns:
-              <input
-                id="number"
-                v-model="props.spans"
-                type="number"
-                :max="(props.breakpoint && BREAKPOINTS[props.breakpoint]?.cols) ?? 20"
-              />
-            </label>
-            <button type="button" @click="handleDeleteModifier">Delete modifier</button>
-          </div>
-          <button type="button" @click="handleAddModifier">Add Grid Modifier</button>
-        </template>
-        <template v-else>
-          <span>Click an existing grid element to change its properties</span>
-        </template>
-      </fieldset>
-    </form>
-    <GridElementsIndicator v-if="gridSettings" :settings="gridSettings" />
-    <main
-      ref="gridElement"
+  <div class="onyx-text playground">
+    <div class="onyx-grid-container">
+      <OnyxHeadline is="h1" class="playground__headline">Grid and breakpoint demo</OnyxHeadline>
+      <OnyxHeadline is="h2">
+        Your current breakpoint:
+        <span class="playground__breakpoint">{{ currentBreakpoint }}</span>
+      </OnyxHeadline>
+
+      <p>
+        Add placeholder components below and link them to the grid properties to get an
+        understanding on how components will behave inside the grid. To see the responsiveness of
+        the onyx grid in action, just use the window resizer to adjust your browser width.
+      </p>
```

Comment:
could you mention that we recommend to use the "Go to full screen" functionality in storybook for the best experience?

![image](https://github.com/user-attachments/assets/4779ba43-d9bf-41ed-886d-4e7fb5f2cc1a)

Reply:
Good idea!